fitbenchmarking/cli/tests/test_checkpoint_handler.py

```diff
@@ -237,13 +262,10 @@ def setUp(self):
           between them
         - Problems in A and C have matching names but not matching data
         """
-        ch_test_files = Path(__file__).parent / 'test_files'
-        self.A = json.load((ch_test_files/'A.json').open()
-                           )['DataSet1']['problems']
-        self.B = json.load((ch_test_files/'B.json').open()
-                           )['DataSet1']['problems']
-        self.C = json.load((ch_test_files/'C.json').open()
-                           )['DataSet1']['problems']
+        ch_test_files = Path(__file__).parent / "test_files"
+        self.A = json.load((ch_test_files / "A.json").open())["DataSet1"]["problems"]
```

Comment:
Originally, flake-8 was 79 and then I changed it to 90 and then checked the default length in ruff. Which is 88 (I believe). I did not have a strong reason to stick with non default value so, I just removed it.

Reply:
No strong opinions from me. We were using the pep8 suggestion https://peps.python.org/pep-0008/#maximum-line-length
I'm not sure it matters too much as long as it's not wildly different from that

Reply:
I will add this line back into the [ruff.lint] settings. The first commit should have this line length. If that looks good then this is an easy one line change.
<img width="266" alt="image" src="https://github.com/user-attachments/assets/8d49ec02-608f-4401-bf33-55f64a23de05">

Reply:
Let's go with 79 because we were not using black and it would be consistent with the original flake8 configuration.

<img width="965" alt="Screenshot 2024-10-11 at 11 47 01" src="https://github.com/user-attachments/assets/24860c8e-c889-4545-9167-8cac7eea4e36">